api/src/metrics/api.rs

```diff
@@ -100,4 +101,16 @@ where
         )
         .await
     }
+
+    async fn events_event_id_get(
+        &self,
+        event_id: String,
+        context: &C,
+    ) -> std::result::Result<EventsEventIdGetResponse, ApiError> {
+        self.record(
+            "/events/{event_id}",
```

Comment:
self.record("/events/",  self.api.events_event_id_get(event_id, context)) -> this will work better with the current prometheus setup?
@dav1do

Reply:
oh dang, I'm sorry. I gave you bad advice.. so I do think /events is nicer, but we already have that for events POST and we don't capture the HTTP method or parameters, so this would be the same 🤦‍♂️ 

This is what the label shows up as, so it would be `/events/{event_id}` if you left it that way. Ideally, we should capture the HTTP method and maybe some path info so we can slice things nicely.

![image](https://github.com/ceramicnetwork/rust-ceramic/assets/5317198/e43d4f3b-d76f-4c05-83de-e364effb7394)